examples/2d/elixir_hyp_diff_nonperiodic.jl

```diff
@@ -0,0 +1,61 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the hyperbolic diffusion equations
+
+resid_tol = 5.0e-12 # TODO: Taal, move this parameter to the callback
+equations = HyperbolicDiffusionEquations2D(resid_tol)
+
+initial_conditions = Trixi.initial_conditions_poisson_nonperiodic
+boundary_conditions = (Trixi.boundary_conditions_poisson_nonperiodic,
```

Comment:
I forget, is there a schematic anywhere of which entry of the four component array corresponds to "bottom" "top" "left" or "right"? This could be useful documentation for new users, or me for that matter :)

Reply:
Good point! We use the usual numbering of the `direction`s, i.e. `1 => -x, 2 => +x, 3 => -y, 4 => +y`. I've added a comment here and started to documents some of these conventions in d8f7b06282e62684ca5b98d3743c05bbd894ebeb